Modules/posixmodule.c

```diff
@@ -9305,6 +9306,7 @@ _pystatvfs_fromstructstatvfs(struct statvfs st) {
     PyStructSequence_SET_ITEM(v, 7, PyLong_FromLong((long) st.f_favail));
     PyStructSequence_SET_ITEM(v, 8, PyLong_FromLong((long) st.f_flag));
     PyStructSequence_SET_ITEM(v, 9, PyLong_FromLong((long) st.f_namemax));
+    PyStructSequence_SET_ITEM(v, 10, PyLong_FromUnsignedLong((unsigned long) st.f_fsid));
```

Comment:
Is the cast to (unsigned long) necessary? Unnecessary casts can mask compiler warnings.

This could also be factored outside the #if block.

Reply:
no, I have only tried to keep the same pattern used for the other items.  I will drop the cast and move the code outside of the #if block.